topdoc/resources/js/docs.js

```diff
@@ -84,16 +84,30 @@ window.addEventListener('click', function(event) {
   }
 });
 
-// Display Slider focus style
-function toggleSliderFocus(event) {
```

Comment:
Was this supposed to be nuked? Seems like you meant to add `toggleInputGroupFocus`, not replace this.

Reply:
`toggleInputGroupFocus` works for Slider as well: https://github.com/adobe/spectrum-css/pull/53/files#diff-85f7fb7fa03b4cdf706f93ebc1b243deR97

Reply:
Godliness.